xcp_d/interfaces/prepostcleaning.py

```diff
@@ -15,12 +15,14 @@ class _removeTRInputSpec(BaseInterfaceInputSpec):
                      mandatory=True,
                      desc=" either bold or nifti ")
     mask_file = File(exists=False, mandatory=False, desc="required for nifti")
-    volumes_to_drop = traits.Int(mandatory=True,
-                                 desc="number of volumes to drop from the beginning")
+    time_todrop = traits.Float(mandatory=True,
```

Comment:
I think we should actually keep this in an integer volumes to drop. We'll calculate this based on the TR in the main code